packages/bruno-app/src/components/RequestPane/HttpRequestPane/index.js

```diff
@@ -176,7 +176,7 @@ const HttpRequestPane = ({ item, collection, leftPaneWidth }) => {
         ) : null}
       </div>
       <section
-        className={classnames('flex w-full flex-1', {
+        className={classnames('flex w-full flex-1 overflow-y-auto max-h-[calc(100vh-210px)]', {
```

Comment:
We should not be using a hardcoded height. This is not maintainable and we will have to change it everytime we make a change that affects this (for instance change height of tabs or top toolbar)